cms/static/js/views/modals/base_modal.js

```diff
@@ -146,7 +150,7 @@ define(["jquery", "underscore", "gettext", "js/views/baseview"],
                 var top, left, modalWindow, modalWidth, modalHeight,
                     availableWidth, availableHeight, maxWidth, maxHeight;
 
-                modalWindow = this.$('.modal-window');
+                modalWindow = this.$el.find(this.options.modalWindowClass)
```

Comment:
Missing semi-colon at the end. Make sure to enable JSHint or JSLint in your editor.